packages/slider/src/component.tsx

```diff
@@ -1,4 +1,10 @@
-import React, { useEffect, useMemo, useRef, useState } from 'react';
```

Comment:
Make sure to run `pnpm lint:check` and `pnpm lint` so that it formats the file correctly. Otherwise you will get a lint-build error :)